draft-ietf-anima-brski-prm.md

```diff
@@ -558,89 +568,95 @@ These endpoints accommodate for the signature-wrapped objects used by BRSKI-PRM
 |------------------------
 | wrappedcacerts | Request CA Certificates    | {{req_cacerts}} |
 |===============
-{: #registrar_ep_table title='Additional Well-Known Endpoints on a BRSKI-PRM Registrar' }
+{: #registrar_ep_table title='Additional Well-Known Endpoints on a BRSKI-PRM Registrar'}
 
 According to {{Section 5.3 of !RFC8995}}, the domain registrar performs the pledge authorization for bootstrapping within his domain based on the Pledge Voucher-Request.
 This behavior is retained in BRSKI-PRM.
 
-The domain registrar MUST possess and trust the IDevID (root or issuing) CA certificate 
-of the pledge vendor/manufacturer.
+The domain registrar MUST possess and trust the IDevID (root or issuing) CA certificate of the pledge vendor/manufacturer.
 
 Further, the domain registrar MUST have its own EE credentials.
```

Comment:
It seems to me as if the Registrar also needs the Registrar-Agent EE certificate to independently fill the field `agent-sign-cert` for the Agent-Proximity Assertion. Otherwise, it just extracts it out of the PVR produced by the pledge... I would assume a comparision of pledge-claimed RegAgt cert and Registrar-knoen RegAgt cert.

Reply:
The Registrar can use the TLS client certificate. Without further text, it would be the only source, but not stated explicitly. Maybe I missed text.

Reply:
Just found the following where the registrar needs to verify the `agent-signed-data`:

> the registrar MAY use the Registrar-Agent EE certificate verified during TLS client authentication;
  otherwise the Registrar-Agent EE certificate(s) need to be provided via configuration or a repository.

Basically, I need the same information again for constructing the RVR.

Maybe it does not hurt to also mention it in Section 6.1 for the Domain Registrar.

Reply:
> It seems to me as if the Registrar also needs the Registrar-Agent EE certificate to independently fill the field `agent-sign-cert` for the Agent-Proximity Assertion. Otherwise, it just extracts it out of the PVR produced by the pledge... I would assume a comparision of pledge-claimed RegAgt cert and Registrar-knoen RegAgt cert.

Yes, the registrar needs to possess the Registrar-Agent-EE certificate, to be able to provide it in the RVR. That said, a statement here would be necessary like:

The domain registrar has access to the Registrar-Agent EE certificate during operation, either by configuration or based on the mutally authenticated TLS communication establishment between the Registrar-Agent and the Registrar. Note that in the latter case, the Registrar is expected to store the received Registrar-Agent EE certificate for later inclusion into PVRs.

Reply:
Okay, I will add a longer explanation to Section 6.1 and have a concise reminder of that at the places where it is used in Section 7.

Reply:
Perfect, thank you

Reply:
Done